cpp/tests/CMakeLists.txt

```diff
@@ -65,29 +65,29 @@ include_directories("${CMAKE_BINARY_DIR}/include"
                     "${CMAKE_SOURCE_DIR}/include"
                     "${CMAKE_SOURCE_DIR}"
                     "${CMAKE_SOURCE_DIR}/src"
-                    "${CMAKE_SOURCE_DIR}/../thirdparty/cub"
-                    "${CMAKE_SOURCE_DIR}/../thirdparty/libcudacxx/include"
-                    "${CMAKE_SOURCE_DIR}/thirdparty/dlpack/include"
+                    "${GDAL_INCLUDE}"
                     "${GTEST_INCLUDE_DIR}"
                     "${RMM_INCLUDE}"
                     "${CUDF_INCLUDE}"
                     "${CUDF_SRC_INCLUDE}"
                     "${CUDF_TEST_INCLUDE}"
-                    "${CUB_INCLUDE}"
+                    "${CUB_INCLUDE_DIR}"
+                    "${THRUST_INCLUDE_DIR}"
+                    "${LIBCUDACXX_INCLUDE_DIR}"
```

Comment:
Do cuspatial tests really need libcudacxx?

Reply:
I believe we do, because we import `cudf/types.hpp`

Reply:
And I remembered that we use timestamps.